package.json

```diff
@@ -51,6 +51,7 @@
     "storybook-addon-intl": "^2.3.0"
   },
   "scripts": {
+    "start-local": "./scripts/serve-local.sh -P",
```

Comment:
Thank you for adding this code, I think it makes more sense from a new user standpoint to use yarn/npm for this.

This script is going to default to pulling data from prod (-P), not sure if this the best way forward. We are creating two ways of using the app, one is through ./scripts/serve-local.sh and another through npm which defaults to prod.

Would there be a way to make prod optional? 

I am totally fine either way 👍

Reply:
Added scripts to point to staging, local, and prod